### ISSUE TITLE
Grant CI/CD principal necessary role(s) to access match APIs

### ISSUE THREAD

**Issue by kingcomma** [18F/piipan#342]
With App Service Authentication implemented after #317, the service principal used by the CI/CD pipeline will need to be granted the necessary role(s) to be able to access the match APIs so it can execute integration tests. This configuration should be part of the IaC (e.g., be configured in `create-service-principal.bash`).

**Comment by kingcomma** [comment]
This issue was superseded by our container-based approach to integration testing (e.g., [`Piipan.Match.Func.Api.IntegrationTests`](https://github.com/18F/piipan/tree/dev/match/tests/Piipan.Match.Func.Api.IntegrationTests)) and our smoke test scripts (e.g., [`test-apim-match-api`](https://github.com/18F/piipan/blob/dev/match/tools/test-apim-match-api.bash)).
